src/Core/Billing/Queries/ISubscriberQueries.cs

```diff
@@ -6,6 +6,18 @@ namespace Bit.Core.Billing.Queries;
 
 public interface ISubscriberQueries
```

Comment:
For this file, I added new methods to retrieve a Stripe `Customer` from an `ISubscriber` safely.